src/main/java/org/springframework/data/web/HateoasSortHandlerMethodArgumentResolver.java

```diff
@@ -56,9 +57,11 @@ public TemplateVariables getSortTemplateVariables(MethodParameter parameter, Uri
 			return TemplateVariables.NONE;
 		}
 
-		String description = String.format("pagination.%s.description", sortParameter);
+		String sortParameterExplode = sortParameter + '*';
```

Comment:
I think we should rather call `TemplateVariable.composite()` on the instance created below.

Reply:
Thank you